Makefile

```diff
@@ -30,7 +33,16 @@ clean-test-leftovers: ## Cleans up any lingering resources in case tests fail ma
 		&>/dev/null || true
 
 .PHONY: test
-test: install-test-deps test-charts test-examples ## Run all chart tests and example tests
+test: unit-test deployment-test ## Run all unit tests and deployment tests
+
+.PHONY: unit-test
+unit-test: README.md ## Run local development tests that must pass prior to submitting a Pull Request
+
```

Comment:
```suggestion
  cd tests/unit; ginkgo

```

Reply:
Seems a bit out-of-scope for the README.md work, isn't it?

Adding https://github.com/spiffe/helm-charts/issues/410 to capture the request.  Resolving this conversation with the addition of that issue.

Reply:
perhaps. but the readme check should be incorporated into the gate in this pr. and then you have to tweak one line either way I think.

Reply:
It's out of scope.  If we don't put some sort of boundary on scope, then eventually anything could go in.  

For example, I would hope that 410 also includes tools to install ginko, and the best time to test those are in 410.

Please proceed past this point, so we can do a "full" and "good" job on the ginko tool chain.